mlrun/model_monitoring/writer.py

```diff
@@ -153,8 +147,41 @@ def _update_kv_db(self, event: _AppResultEvent) -> None:
             key=app_name,
             attributes=event,
         )
+        if endpoint_id not in self._kv_schemas:
+            self._generate_kv_schema(endpoint_id)
         logger.info("Updated V3IO KV successfully", key=app_name)
 
+    def _generate_kv_schema(self, endpoint_id: str):
+        """Generate V3IO KV schema file which will be used by the model monitoring applications dashboard in Grafana."""
+        fields = [
+            {"name": WriterEvent.APPLICATION_NAME, "type": "string", "nullable": False},
+            {"name": WriterEvent.SCHEDULE_TIME, "type": "string", "nullable": False},
+            {"name": WriterEvent.RESULT_NAME, "type": "string", "nullable": False},
+            {"name": WriterEvent.RESULT_KIND, "type": "double", "nullable": False},
+            {"name": WriterEvent.RESULT_VALUE, "type": "double", "nullable": False},
+            {"name": WriterEvent.RESULT_STATUS, "type": "double", "nullable": False},
```

Comment:
why result-status is type "double"?

Reply:
because it's a numeric value defined under an `IntEnum` class called `ResultStatusApp`:

```
class ResultStatusApp(enum.IntEnum):
    """
    Enum for the result status values, detected means that the app detected some problem.
    """

    irrelevant = -1
    no_detection = 0
    potential_detection = 1
    detected = 2
```

Reply:
So why not `"integer"` instead of `"double"`?